common/floppy_config.go

```diff
@@ -38,7 +38,7 @@ func (c *FloppyConfig) Prepare(ctx *interpolate.Context) []error {
 	}
 
 	for _, path := range c.FloppyDirectories {
-		if strings.IndexAny(path, "*?[") >= 0 {
+		if strings.ContainsAny(path, "*?[") {
```

Comment:
My only concern would be are methods like ContainsAny available in later version of Go only, and if so, does that impact Packer's ability to compile against older versions.  What is the minimum version that Packer targets?  We typically build with latest - 1 (1.7).